rio_tiler/utils.py

```diff
@@ -274,6 +274,19 @@ def linear_rescale(
             returns rescaled image array.
 
     """
+    if in_range is None:
+        in_range = (image.min(), image.max())
+        warnings.warn(
+            f"No input range set, using image min/max values ({in_range[0], in_range[1]})",
+            UserWarning,
+        )
+
+    if out_range is None:
+        warnings.warn(
+            "No output range set, using image (0, 255)", UserWarning,
+        )
+        out_range = (0, 255)
+
```

Comment:
Needed update! 

- we now don't set default input range to 0,1 but to min,max of the input image! 
- The output default range is also `fixed` to be `0, 255` not `1,255`

Reply:
You could also just require the input range to be provided...? Because the local min/max isn't the same as the global min/max

Reply:
yes, I was worry about breaking more this function because it always had defaults. 

about min/max, the linear rescale could be use on `preview` which would be a valid case where to use min/max of the array.